ps_sharebuttons.php

```diff
@@ -206,6 +208,22 @@ public function getWidgetVariables($hookName, array $params)
             );
         }
 
+        if (Configuration::get('PS_SC_VKONTAKTE')) {
+            $social_share_links['vkontakte'] = array(
+                'label' => $this->trans('VKontakte', array(), 'Modules.Sharebuttons.Shop'),
+                'class' => 'vkontakte',
+                'url' => 'http://vk.com/share.php?url='.$sharing_url,
```

Comment:
Might need to use https instead of http here

Reply:
Sure. Fixed